frontend/test/metabase/lib/expressions/completer.unit.spec.js

```diff
@@ -0,0 +1,31 @@
+import { partialMatch } from "metabase/lib/expressions/completer";
+
+describe("metabase/lib/expressions/completer", () => {
+  describe("partialMatch", () => {
+    it("should get the function name", () => {
+      expect(partialMatch("Lowe")).toEqual("Lowe");
+      expect(partialMatch("NOT (ISNULL")).toEqual("ISNULL");
+    });
+
+    it("should get the field name", () => {
+      expect(partialMatch("[Deal]")).toEqual("[Deal]");
+      expect(partialMatch("A")).toEqual("A");
+      expect(partialMatch("B AND [Ca")).toEqual("[Ca");
+      expect(partialMatch("[Sale] or [Good]")).toEqual("[Good]");
+      expect(partialMatch("[")).toEqual("[");
+    });
+
+    it("should ignore operators and literals", () => {
+      expect(partialMatch("X OR")).toEqual(null);
+      expect(partialMatch("42 + ")).toEqual(null);
```

Comment:
Shouldn't it test the operator, not the space?
```suggestion
      expect(partialMatch("42 +")).toEqual(null);
```

Reply:
Good catch, thanks!! I've changed it accordingly.